.agents/automation/reports/database-migration-consistency.md

```
@@ -1,3 +1,45 @@
-# Database Migration Consistency Auditor
+# Database Migration Consistency Audit Report
 
-No automation execution has been recorded yet.
+## Purpose
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)